controllers/DocumentController.php

```diff
@@ -8,7 +8,7 @@
 // Copyright © - INRA - 2017
 // Creation date: June 2017
 // Contact: morgane.vidal@inra.fr, anne.tireau@inra.fr, pascal.neveu@inra.fr
-// Last modification date:  June, 2017
+// Last modification date:  Jan., 2019
```

Comment:
Cold you remove on the files you modify 'last modification date and subject"
thanks